ajax/libs/planout/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "planout",
+  "filename": "planout.min.js",
+  "version": "3.0.1",
+  "description": "PlanOut is a framework and programming language for online field experimentation",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/HubSpot/PlanOut.js"
+  },
+  "keywords": [
+    "experiments",
+    "a/b testing",
+    "multivariate testing",
+    "experiment design",
+    "ab",
+    "split testing"
+  ],
+  "author": "Guy Aridor",
+  "license": "BSD-3-clause",
```

Comment:
`BSD-3-Clause`, you should see warning from the `fixFormat` tool.

cc @cdnjs/intern2 @cdnjs/intern3